src/Atmos/Model/tendencies_energy.jl

```diff
@@ -40,7 +52,7 @@ struct DiffEnthalpyFlux{PV <: Energy} <: TendencyDef{Flux{SecondOrder}, PV} end
 function flux(::DiffEnthalpyFlux{Energy}, atmos, args)
     @unpack state, diffusive = args
     @unpack D_t = args.precomputed.turbulence
-    d_h_tot = -D_t .* diffusive.∇h_tot
+    d_h_tot = -D_t .* diffusive.energy.∇h_tot
```

Comment:
I didn't see where `energy.∇h_tot` is calculated. For `theta_li`, it's important that this is `theta_li` itself, not some other (enthalpy) variable. Is it?

Reply:
It's computed [here](https://github.com/CliMA/ClimateMachine.jl/pull/1935/files#diff-4032d710c4d46f450b770888b8ae491bd3f2134e6b1299a21170040198b070e5R52), for the energy formulation only.